lab-steven/readme.md

```diff
@@ -0,0 +1,25 @@
+# Lab 24 - ngAdventure
+
+#### Description
+
+
```

Comment:
please add the difference between a controller and a service in your docs per lab instructions